src/Session/AdapterPluginManagerDelegatorFactory.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Laminas\Cache\Storage\Adapter\Session;
 
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
It seems cs issue

Reply:
Good catch, in other adapters I added conflict with old servicemanager versions and moved to PSR instead.